src/market/demand.ts

```diff
@@ -27,6 +27,7 @@ export interface DemandOptions {
   offerFetchingInterval?: number;
   proposalTimeout?: number;
   eventTarget?: EventTarget;
+  startupTimeout?: number;
```

Comment:
This field should be documented as well.

Reply:
this is unnecessary, left by mistake, i will delete it